src/openforms/submissions/admin.py

```diff
@@ -498,8 +498,16 @@ class SubmissionReportAdmin(PrivateMediaMixin, admin.ModelAdmin):
         "submission__public_registration_reference",
     )
     raw_id_fields = ("submission",)
+    fields = (
+        "title",
+        "content",
+        "submission",
+        "last_accessed",
+        "task_id",
+    )
```

Comment:
why do you specify these explicitly? by default all model fields are used iirc. I'd expect this not to be necessary.